_posts/2021-12-14-upcoming-filesystem-encryption-integrity.adoc

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "Upcoming filesystem realm encryption and integrity support in Elytron"
+date: 2021-12-14
+tags:
```

Comment:
@Ashpan Just minor, but we could add filesystem-realm tag now, and maybe encryption as well

Reply:
Ah good catch, just added